taui/src/actions/listings.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import fetch from '@conveyal/woonerf/fetch'
+
+import {fwdGeocodeBatch} from '../utils/fwd-geocode'
+
+import {addActionLogItem} from './log'
+
+const REALTOR_BASE_URL = 'https://akk8p5k8o0.execute-api.us-east-1.amazonaws.com/staging/get-realtor-listings'
+const BHA_BASE_URL = 'https://akk8p5k8o0.execute-api.us-east-1.amazonaws.com/staging/-get-bha-listings?'
+const REALTOR_TYPE = 'set Realtor listings'
```

Comment:
```suggestion
const REALTOR_ACTION_TYPE = 'set Realtor listings'
```
I think this (and `BHA_TYPE`) would be more self-explanatory if they call out that they're an action type; when I first read this I wasn't 100% sure what type of type these were.